core/domain/stats_services.py

```diff
@@ -1124,6 +1124,87 @@ def get_learner_answer_details(entity_type, state_reference):
     return None
 
 
+def get_learner_answer_info_for_exploration(exp_id):
+    """Returns the learner answer info in dict format.
+
+    Args:
+        exp_id: str. The ID of the exploration.
+
+    Returns:
+        list. Consists of dicts
+            with state name as keys, and a list of learner answer info dicts as
```

Comment:
Why break after 'dicts'? Please join the start of this line to the end of the preceding one and break only if the line would exceed 80 chars otherwise.

Reply:
Made services to return domain object only

Reply:
Sorry -- I think you might be addressing the wrong comment? This one is about linting.

Reply:
I have removed the service in which this docstring was.